google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/prepare/datacatalog_tag_factory.py

```diff
@@ -33,19 +33,45 @@ def make_tag_for_app(self, tag_template, app_metadata):
 
         self._set_string_field(tag, 'id', app_metadata.get('id'))
 
+        owner = app_metadata.get('owner')
+        if owner:
+            owner_user_dir = owner.get('userDirectory')
+            owner_user_id = owner.get('userId')
+
+            if owner_user_dir and owner_user_id:
+                self._set_string_field(tag, 'owner_username',
+                                       f'{owner_user_dir}\\\\{owner_user_id}')
+
+            self._set_string_field(tag, 'owner_name', owner.get('name'))
+
+        self._set_string_field(tag, 'modified_by_username',
+                               app_metadata.get('modifiedByUserName'))
+
         self._set_timestamp_field(
             tag, 'publish_time',
             datetime.strptime(app_metadata.get('publishTime'),
                               self.__INCOMING_TIMESTAMP_UTC_FORMAT))
 
         self._set_bool_field(tag, 'published', app_metadata.get('published'))
 
+        self._set_timestamp_field(
+            tag, 'last_reload_time',
+            datetime.strptime(app_metadata.get('lastReloadTime'),
```

Comment:
I haven't seen this scenario in the tests, but since the `lastReloadTime` is optional, [according to the docs](https://help.qlik.com/en-US/sense-developer/November2020/APIs/RepositoryServiceAPI/index.html?page=1542), I've added a nullity check to avoid problems.